Assets/BossRoom/Scripts/Client/Audio/ClientMusicPlayer.cs

```diff
@@ -51,7 +51,7 @@ private void PlayTrack(AudioClip clip, bool looping, bool restart)
             if (m_source.isPlaying)
             {
                 // if we dont want to restart the clip, do nothing if it is playing
-                if (!restart && m_source.clip == clip) { return; }
+                if (!restart && m_source.clip==clip) { return; }
```

Comment:
Can we do a separate pass that does automatic formatting of all files using the same format.  There are some places (see ActionFX.cs) where it's adding spaces inside of parethesese but not removing spaces around operators, which seems to be the opposite of this.  It clutters up this PR quite a bit.